Cargo.toml

```diff
@@ -39,13 +39,24 @@ members = [
 # Ignore bindings stuff given they are special snowflakes that require exotic toolchain and dependencies
 default-members = ["oxidation/libparsec"]
 
-# A custom profile that is the fusion of `release` & `debug` profile.
-# We want the speed of the `release` profile but with the debug assertions of the `debug` profile.
-[profile.integration]
-inherits = "release"
-overflow-checks = true
-debug-assertions = true
-debug = true
-incremental = true
+# Rust unoptimized code is so slow that we customize dev profile to enable some
+# optimizations, otherwise Python tests take forever.
+[profile.dev]
+opt-level = 1
```

Comment:
not required with the profile `ci-python`

Reply:
This is to have incremental build and debug info present when doing dev in local